source/hp/dof_handler.cc

```diff
@@ -959,6 +959,13 @@ namespace hp
 
 
 
+  template <int dim, int spacedim>
+  DoFHandler<dim,spacedim>::DoFHandler ()
+    :
+    tria(nullptr, typeid(*this).name()),
+    faces (nullptr)
+  {}
+
   template <int dim, int spacedim>
```

Comment:
Total nitpick, but we like to have three empty lines between each two function definitions.